docs/src/main/sphinx/security/ldap.rst

```diff
@@ -109,6 +109,8 @@ Property                           Description
 ``ldap.ignore-referrals``          Ignore referrals to other LDAP servers while
                                    performing search queries. Defaults to ``false``.
 ``ldap.cache-ttl``                 LDAP cache duration. Defaults to ``1h``.
+``ldap.timeout.connection``        Timeout for establishing an LDAP connection.
+``ldap.timeout.read``              Timeout for reading data from an LDAP connection.
```

Comment:
`ldap.read-timeout` and `ldap.connection-timeout`. That way is more common in Trino.